pkg/storage/storagebackend/config.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package storagebackend
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/storage"
+)
+
+const (
+	StorageTypeUnset = ""
+	StorageTypeETCD2 = "etcd2"
+	StorageTypeETCD3 = "etcd3"
+)
+
+// Config is configuration for creating a storage backend.
+type Config struct {
+	Type                     string
+	Codec                    runtime.Codec
+	Prefix                   string
+	ServerList               []string
+	KeyFile                  string
+	CertFile                 string
+	CAFile                   string
+	Quorum                   bool
+	DeserializationCacheSize int
+}
+
+// Create creates a storage backend based on given config.
+func Create(c Config) (storage.Interface, error) {
+	switch c.Type {
+	case StorageTypeUnset, StorageTypeETCD2:
+		return newETCD2Storage(c)
+	case StorageTypeETCD3:
+		return newETCD3Storage(c)
+	default:
+		return nil, fmt.Errorf("unknown storage type: %s", c.Type)
```

Comment:
The answer is: [glog.Fatalf](https://github.com/kubernetes/kubernetes/blob/c408714b76de401d0b3b2725400ba8d3074f4293/cmd/kube-apiserver/app/server.go#L212-L214)